test-services/check-links/config.json

```diff
@@ -1,7 +1,7 @@
 {
     "ignorePatterns": [
         {
-            "pattern": "^https://(dl|ingest|api)\\.signalfx\\.com"
+            "pattern": "^https://.*\\.signalfx\\.com"
```

Comment:
What about links to docs.signalfx.com that might be invalid?

Reply:
> What about links to docs.signalfx.com that might be invalid?

Ah, thanks, forgot about that. Will change.